### PR TITLE
bug #13343 : AFP Binding Error

### PR DESCRIPTION
**Pull request by DeepikaDhiman**
Changed the Services/AFP/Bind IP formfield as CIFS formfield,
now user will be able to bind the lagg Interface IP with AFP.

### CODE REVIEW

File: gui/services/forms.py [hunk 1]
```diff
@@ -132,6 +132,11 @@ def save(self, *args, **kwargs):
             else:
                 started = _notifier.stop("domaincontroller")
 
+        elif obj.srv_service == 'afp' and _notifier._started_domaincontroller():
+            obj.srv_enable = True
+            obj.save()
+            started = True
+
```
Comment (williambr) [inline gui/services/forms.py]: Why did you have to change this?

Comment (DeepikaDhiman) [inline gui/services/forms.py]: I just tried implementing AFP service as CIFS, however testing the code without these lines also work, So they aren't required. I'll make the required changes.

File: gui/services/models.py [hunk 2]
```diff
@@ -318,8 +318,7 @@ class AFP(Model):
         ),
         max_length=255,
         blank=True,
-        choices=list(choices.IPChoices()),
-        default='',
+        null=True,
```
Comment (williambr) [inline gui/services/models.py]: null=True, why do you need this?
Have you tested only default='' ?

Comment (DeepikaDhiman) [inline gui/services/models.py]: Since,I wanted the bindip field in AFP to be similar to that of CIFS, I implemented it same as CIFS, 
But yeah, "default=' ' " would work same as "null=True".
I chose "null=True", since this is how it is implemented for CIFS bindip.

Comment (williambr) [inline gui/services/models.py]: If you change it to null it will require a migration. If it is not needed it should be changed to default='' instead of null=True.

Comment (DeepikaDhiman) [inline gui/services/models.py]: If it requires the creation of migration file, then I think I should  just revert this change.
Thanks for correcting me here.

**Comment by williambr** [comment]
Merged. As always, set redmine ticket to Build Testing. Thank you

**Comment by DeepikaDhiman** [comment]
Thanks William
